pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -1336,8 +1336,9 @@ type LinuxContainerSecurityContext struct {
 	NamespaceOptions *NamespaceOption `protobuf:"bytes,3,opt,name=namespace_options,json=namespaceOptions" json:"namespace_options,omitempty"`
 	// Optional SELinux context to be applied.
 	SelinuxOptions *SELinuxOption `protobuf:"bytes,4,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
-	// The user to run the the container process as, it could be uid or user
-	// name.
+	// User to run the entrypoint of the sandbox process. Can be either UID or
```

Comment:
Shouldn't "sandbox process" be "container entrypoint" since this is part of generic containerconfigs, not just pod sandbox?